amti/clis/create.py

```diff
@@ -29,7 +29,16 @@
     '--live', '-l',
     is_flag=True,
     help='Create HITs on the live MTurk site.')
-def create_batch(definition_dir, data_path, save_dir, live):
+@click.option(
+    '--verify',
+    is_flag=True,
+    help='Always verify costs.')
+@click.option(
+    '--force', '-f',
+    is_flag=True,
+    help='Do not verify hit costs.')
```

Comment:
Rather than an additional `--force` option, this seems like a great place to use Click's [Boolean flags feature](https://click.palletsprojects.com/en/7.x/options/#boolean-flags). So instead we'd have:

```python
@click.option(
    '--check-cost/--no-check-cost',
    '-c/-n',
    default=True,
    help='Whether to prompt the user to check the batch's cost.')
```

I also changed the name from `--verify` to `--check-costs` since then we won't clash with the `--verbose` flag when using abbreviated versions.

Reply:
Didn't know about this feature! makes much more sense this way :)